modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
+
+*Units*: bytes per second.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `104857600`
+
+---
+
+=== raft_max_concurrent_append_requests_per_follower
+
+Maximum number of concurrent append entries requests sent by leader to one follower.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`0`, `4294967295`]
+
+*Default:* `16`
+
+---
+
+=== raft_max_recovery_memory
+
+Max memory that can be used for reads in raft recovery process by default 15% of total memory.
+
+*Requires Restart:* No
+
+*Nullable:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `null`
+
+---
+
+=== raft_recovery_concurrency_per_shard
+
+How many partitions may simultaneously recover data to a particular shard. This is limited to avoid overwhelming nodes when they come back online after an outage.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `64`
+
+---
+
+=== raft_recovery_default_read_size
+
+Default size of read issued during raft follower recovery.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `524288`
+
+---
+
+=== raft_recovery_throttle_disable_dynamic_mode
+
+Disables dynamic rate allocation in recovery throttle (advanced).
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== raft_replica_max_pending_flush_bytes
+
+Max not flushed bytes per partition. If configured threshold is reached log will automatically be flushed even though it wasn't explicitly requested.
+
+*Requires Restart:* No
+
+*Nullable:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `262144`
+
+---
+
+=== raft_replicate_batch_window_size
+
+Max size of requests cached for replication.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `1048576`
+
+---
+
+=== raft_smp_max_non_local_requests
+
+Maximum number of x-core requests pending in Raft seastar::smp group. (for more details look at `seastar::smp_service_group` documentation).
+
+*Requires Restart:* No
+
+*Nullable:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`0`, `4294967295`]
+
+*Default:* `null`
+
+---
+
+=== raft_timeout_now_timeout_ms
+
+Timeout for a timeout now request.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `1s`
+
+---
+
+=== raft_transfer_leader_recovery_timeout_ms
+
+Timeout waiting for follower recovery when transferring leadership.
```

Comment:
Use to specify the follower recovery timeout waiting period when transferring leadership.